stetho-sample/src/main/java/com/facebook/stetho/sample/APODActivity.java

```diff
@@ -109,7 +109,7 @@ public void bindView(View view, Context context, Cursor cursor) {
       int bindPosition = cursor.getPosition();
       holder.position = bindPosition;
 
-      final String imageUrl = cursor.getString(APODPostsQuery.DESCRIPTION_IMAGE_URL_INDEX);
+      final String imageUrl = cursor.getString(APODPostsQuery.LARGE_IMAGE_URL_INDEX);
```

Comment:
You mean `imageUrl` can now be null?  It could be null before too, technically, it just isn't in practice.  Also, I checked and I don't think the code will necessary do the wrong thing if it's null since the Networker will just fail the request immediately.  I haven't tested that, though.